developer-guide.md

```diff
@@ -31,7 +31,7 @@ You can get started experimenting with the new feature on `localhost` either by
 An exploration of the motivations and reasoning behind origin trials is provided in [the explainer](explainer.md). The TL;DR is that we strongly value the feedback of real web developers (that means you!) during the process of designing and standardizing new features. We believe origin trials provide a good way of encouraging that feedback, while being extremely careful that the experiments aren’t used by sites in production-critical roles or as if they’re finalized features.
 
 ## What experimental features are currently available?
-The [experimental feature tracker](available-trials.md) lists all of the currently available features and their experiment timelines.
+The [experimental feature tracker](https://developers.chrome.com/origintrials/#/trials/active) lists all of the currently available features.
```

Comment:
We might as be consistent in how we refer to the console. I think "experimental feature tracker" makes it sound like a different thing entirely.

```suggestion
The [developer console](https://developers.chrome.com/origintrials/#/trials/active) lists all of the currently available features.
```